cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -58,7 +58,7 @@ <h3 class="answer-snippet">
                 <strong>
                     Answer:
                 </strong>
-                {{ page.snippet | safe }}
+                {{ page.snippet | richtext | safe }}
```

Comment:
`richtext` and `safe` are redundant I believe? I think you only need `richtext`.

Reply:
Double-check with a bewd.

Reply:
Good point, @anselmbradford! I think I'll merge this now to fix the issue, but I'll address the redundancy in the next PR. Thanks!